flatpak-tweaks.patch

```diff
@@ -0,0 +1,176 @@
+commit a50acc67521dd1a3f28ad523c05d5328835a496f
```

Comment:
Bear with me – this will be part of the next release?

Reply:
Here it is: https://github.com/contour-terminal/contour/pull/767